osf/migrations/0085_migrate_collections_data.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-03-05 20:17
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('osf', '0084_pre_migrate_collections'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            [
+                """
+                -- Copy all existing collections into new table, keeping old pks
```

Comment:
I read over the comments and it makes sense, but only skimmed the SQL.

Reply:
💯 🅰️➕

I've verified it works and maintains data-integrity (at least, for a small pseudo-random sample) for prod.